.circleci/config.yml

```diff
@@ -70,7 +70,6 @@ jobs:
       - run: |
           set -euo pipefail
           source server/scripts/_constants.env
-          docker build -t "${IMAGE_ORG_AND_REPO}:dev" -f server/Dockerfile server/
```

Comment:
no longer needed thanks to `ImageBuildSpec`

Reply:
well done!